ads/tests/test_search.py

```diff
@@ -207,10 +214,31 @@ def test_init(self):
             'aff:"institute" star'.split(),
         )
 
-        sq = SearchQuery(q="q", token="test-token")
+        sq = SearchQuery(q="star", token="test-token")
         self.assertEqual(sq.token, "test-token")
         self.assertEqual(sq._token, "test-token")
 
+        sq = SearchQuery(q="star", sort="date")
+        self.assertEqual(sq.query['cursorMark'], '*')
+        self.assertNotIn('start', sq.query)
+        self.assertEqual(sq.query['sort'], 'date desc,id desc')
+
+        sq = SearchQuery(q="star", start=0)
+        self.assertEqual(sq.query['start'], 0)
+        self.assertEqual(sq.query['sort'], "score desc")
+
```

Comment:
How about adding `self.assertNotIn('cursorMark', sq.query)` just for consistency with line 223.